Added count() method to utils.js.

szymonkups opened the pull request:
Fixes #10.

pjasiun commented on src/utils.js:
```diff
@@ -176,6 +176,24 @@ const utils = {
+	 * Returns the number of items return by the iterator.
+	 *
+	 *		utils.count( [ 1, 2, 3, 4, 5 ] ); // 5;
+	 *
```
@memberOf utils.utils

Reinmar commented:
We already have such method in the test utils: https://github.com/ckeditor/ckeditor5-utils/blob/master/tests/_utils/utils.js#L21

szymonkups commented:
Fixed. It was used only for tests, then this method started to be needed also in development. I've removed `getIteratorCount` method from test utils, and moved it as `count()` to utilities.

Reinmar commented:
Please also remember about doing this change in the core (I think that no other repo uses this method).

pjasiun commented on tests/utils.js:
```diff
@@ -6,9 +6,8 @@
-import utilsTestUtils from '/tests/utils/_utils/utils.js';
 
-const getIteratorCount = utilsTestUtils.getIteratorCount;
+const getIteratorCount = utils.count;
```
Wouldn't it be better to use: `const count = utils.count;`? `getIteratorCount` was there because of the name of the method and now it's messy.
